core/commons/che-core-commons-observability/src/main/java/org/eclipse/che/commons/observability/CountedRejectedExecutionHandler.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.commons.observability;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+import java.util.concurrent.RejectedExecutionHandler;
+import java.util.concurrent.ThreadPoolExecutor;
+
+/**
+ * Wrapper of {@link RejectedExecutionHandler} that reports to {@link MeterRegistry} number of
+ * rejections.
+ */
+public class CountedRejectedExecutionHandler implements RejectedExecutionHandler {
```

Comment:
Can [that](https://github.com/eclipse/che/blob/master/core/che-core-metrics-core/src/main/java/org/eclipse/che/core/metrics/ExecutorServiceMetrics.java#L82) class be replaced with this new one ?

Reply:
yes. I've removed unused https://github.com/eclipse/che/blob/master/core/che-core-metrics-core/src/main/java/org/eclipse/che/core/metrics/ExecutorServiceMetrics.java#L82 class.